Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewRenderer.cs

```diff
@@ -163,7 +164,7 @@ static ListViewBase CreateHorizontalListView()
 					(ItemsPanelTemplate)Windows.UI.Xaml.Application.Current.Resources["HorizontalListItemsPanel"]
 			};
 
-			ScrollViewer.SetHorizontalScrollMode(horizontalListView, ScrollMode.Auto);
+			ScrollViewer.SetHorizontalScrollMode(horizontalListView, WScroolMode.Auto);
```

Comment:
These changes do not affect the `UWP` and `CollectionVIew`. They are made only to avoid conflicting class `ScrollMode` between `Windows.UI.Xaml.Controls` and `Xamarin.Forms` namespaces.